scriptmodules/libretrocores/lr-fba-next.sh

```diff
@@ -52,6 +52,9 @@ function configure_lr-fba-next() {
     ensureSystemretroconfig "fba"
     ensureSystemretroconfig "neogeo"
 
+    # Create samples directory
+    mkUserDir "$biosdir/fba/samples/"
```

Comment:
mkUser dir will only chown the lowest directly, so as fba may not exist you need to create that first, or chown "$biosdir/fba" manually.